app/workers/publishing_api_gone_worker.rb

```diff
@@ -1,7 +1,9 @@
 class PublishingApiGoneWorker < PublishingApiWorker
-  def perform(content_id, locale)
+  def perform(content_id, alternative_path, explanation, locale)
```

Comment:
This will change the signature for inflight jobs during deployment. We could support this by something like:
```ruby
def perform(content_id, options = {})
  if options.is_a?(String)
    options = {locale: options}
  end
  # stuff
end